src/ubuntu/18.04/helix/arm64v8/Dockerfile

```diff
@@ -50,19 +50,15 @@ RUN ln -sf /usr/bin/python3 /usr/bin/python && \
     export CRYPTOGRAPHY_DONT_BUILD_RUST=1 && \
     pip install ./helix_scripts-*-py3-none-any.whl
 
-# build MsQuic as we don't have packages
-RUN cd /tmp && \
-    mkdir pwsh && \
-    cd pwsh && \
-    curl -LO https://github.com/PowerShell/PowerShell/releases/download/v7.2.5/powershell-7.2.5-linux-arm64.tar.gz && \
-    echo "709265A0B99232CD8AB6F8A02C01F3AEE94262B959E1A4B0FD871C8789C07396 powershell-7.2.5-linux-arm64.tar.gz" | sha256sum --check - && \
-    tar xf powershell-7.2.5-linux-arm64.tar.gz && \
-    cd .. && \
-    git clone --depth 1 --single-branch --branch main --recursive https://github.com/dotnet/msquic && \
-    cd msquic/src/msquic  &&  PATH=~/.dotnet/tools:$PATH /tmp/pwsh/pwsh scripts/build.ps1 -arch arm64 -Config Release -UseSystemOpenSSLCrypto -DisableTools -DisableTest -DisablePerf && \
-    cp artifacts/bin/linux/arm64_Release_openssl/libmsquic.so.2 artifacts/bin/linux/arm64_Release_openssl/libmsquic.lttng.so.2.2.0 /usr/lib/aarch64-linux-gnu && \
-    cd /tmp && \
-    rm -r pwsh msquic
+# Add MsQuic
+RUN curl -LO https://packages.microsoft.com/keys/microsoft.asc && \
+    echo 2cfd20a306b2fa5e25522d78f2ef50a1f429d35fd30bd983e2ebffc2b80944fa microsoft.asc| sha256sum --check - && \
+    apt-key add microsoft.asc && \
+    rm microsoft.asc && \
+    apt-add-repository https://packages.microsoft.com/ubuntu/18.04/prod && \
+    apt-get update && \
+    apt-get install -y libmsquic && \
```

Comment:
Build is getting an error, not finding this package.

Reply:
I will check. The 2.1.1 (and earlier) seems to be there... https://packages.microsoft.com/ubuntu/18.04/prod/pool/main/libm/libmsquic/
(and I can follow-up with msquic)

Reply:
oh. arm. it its not there.... ;(

Reply:
reported to msquic and I put in workaround with comment for now.